tests/acceptance/features/bootstrap/Sharing.php

```diff
@@ -3479,6 +3479,47 @@ public function assertNoSharesOfUserAreInState(string $user, string $state):void
 		);
 	}
 
+	/**
+	 * @Given /^user "([^"]*)" has unshared folder "([^"]*)" for "([^"]*)"$/
```

Comment:
Should it be `from` ? I'm not so sure 
```suggestion
	 * @Given /^user "([^"]*)" has unshared folder "([^"]*)" from "([^"]*)"$/
```
cc @phil-davis